lib/vmdb/loggers.rb

```diff
@@ -54,29 +54,29 @@ def self.create_logger(log_file_name, logger_class = ManageIQ::Loggers::Base)
     end
 
     private_class_method def self.create_container_logger(log_file_name, logger_class)
-      return nil unless (logger = create_raw_container_logger)
+      return nil unless (logger = create_raw_container_logger(log_file_name))
 
       create_wrapper_logger(log_file_name, logger_class, logger)
     end
 
-    private_class_method def self.create_raw_container_logger
+    private_class_method def self.create_raw_container_logger(log_file_name)
       return unless ENV["CONTAINER"]
 
       require "manageiq/loggers/container"
-      ManageIQ::Loggers::Container.new
+      ManageIQ::Loggers::Container.new($stdout, :progname => log_file_to_prog_name(log_file_name))
```

Comment:
I don't think you need stdout here. We default it on the other side, and the progname is a kwarg.

```suggestion
      ManageIQ::Loggers::Container.new(:progname => log_file_to_prog_name(log_file_name))
```

Reply:
:+1: fixed